librz/analysis/esil/esil.c

```diff
@@ -916,7 +895,7 @@ static bool esil_xoreq(RzAnalysisEsil *esil) {
 			esil->lastsz = esil_internal_sizeof_reg(esil, dst);
 			ret = rz_analysis_esil_reg_write(esil, dst, num ^ num2);
 		} else {
-			ERR("esil_xoreq: empty stack");
+			ESIL_LOG("esil_xoreq: empty stack\n");
```

Comment:
I wonder if its better to use `RZ_LOG_WARN()` instead of `ESIL_LOG()`

Reply:
ESIL_LOG is called only when `esil->verbose` is invoked. since it is normally set to `false` and we are also removing slowly esil, it won't matter much